src/pybind/mgr/snap_schedule/module.py

```diff
@@ -6,7 +6,7 @@
 import errno
 import json
 import sqlite3
-from typing import Any, Dict, Optional, Tuple
+from typing import Any, Dict, Optional, Tuple, List
```

Comment:
missed to undo this import. Otherwise, looks good to me.

Reply:
removed reference to unused import `List`